webpack.config.ts

```diff
@@ -96,7 +95,6 @@ const config: Configuration = {
 		]
 	},
 	plugins: [
-		new ESBuildPlugin(),
```

Comment:
?

Is it picked up automatically now? I see the loader still in the dependencies

Reply:
We still use the loader, just there is no need to do `new ESBuildPlugin()`